src/assets/monitoring_locations/delawareBasinNextGenerationLocationsSorted.js

```diff
@@ -0,0 +1,1098 @@
+            },
+            {
+                "type": "Feature",
+                "properties": {
+                    "site_id": "USGS-01427500",
+                    "latitude": 41.76055556,
+                    "longitude": -75.04838889,
+                    "new_gage": true,
+                    "enhanced_gage": false,
+                    "temperature": false,
+                    "specific_conductance": false,
+                    "camera": false,
+                    "flyToCommands": {
+                        "bearing": 350,
+                        "center": [-75.04838889, 41.76055556],
+                        "zoom": 8,
+                        "pitch": 90,
+                        "speed": 0.2,
+                        "curve": 1,
+                        "essential": true
+                    },
+                },
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [-75.04838889, 41.76055556]
+                }
+            },
+            {
+                "type": "Feature",
+                "properties": {
+                    "site_id": "USGS-01427510",
+                    "latitude": 41.75675,
+                    "longitude": -75.0574167,
+                    "new_gage": false,
+                    "enhanced_gage": true,
+                    "temperature": false,
+                    "specific_conductance": false,
+                    "camera": false,
+                    "flyToCommands": {
+                        "bearing": 350,
+                        "center": [-75.0574167, 41.75675],
+                        "zoom": 8,
+                        "pitch": 90,
+                        "speed": 0.2,
+                        "curve": 1,
+                        "essential": true
+                    },
+                },
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [-75.0574167, 41.75675]
+                }
+            },
+            {
+                "type": "Feature",
+                "properties": {
+                    "site_id": "USGS-01428000",
+                    "latitude": 41.56388889,
+                    "longitude": -75.0144444,
+                    "new_gage": true,
+                    "enhanced_gage": false,
+                    "temperature": false,
+                    "specific_conductance": false,
+                    "camera": false,
+                    "flyToCommands": {
+                        "bearing": 350,
+                        "center": [-75.0144444, 41.56388889],
+                        "zoom": 8,
+                        "pitch": 90,
+                        "speed": 0.2,
+                        "curve": 1,
+                        "essential": true
+                    },
+                },
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [-75.0144444, 41.56388889]
+                }
+            },
+            {
```

Comment:
@mwernimont i completely missed this location the first time somehow, it goes in your file...

Reply:
USGS-01428500